docker-storage-setup.sh

```diff
@@ -720,6 +719,21 @@ unit: sectors
 
 ${dev}1 : start=     2048, size=  ${size}, Id=8e
 EOF
+}
+
+create_partition_parted(){
+  local dev="$1"
+  parted $dev --script mklabel dos mkpart primary 0% 100%
```

Comment:
I was following your comment for `mklabel dos` :)
https://github.com/projectatomic/docker-storage-setup/pull/191#discussion_r98042388

I found the `mkpart primary 0% 100%` syntax on stack overflow. I tested it and it works. We can also do 0 and -1, that should work too.